Automattic/jetpack
# VideoPress: add Share and Allow Download option to dashboard video detail page

Fixes #28702 

## Proposed changes:

* Add Share and Allow Download option to dashboard video detail page:
![image](https://user-images.githubusercontent.com/16329583/217271334-953a1a35-6d35-4b12-8c42-19b0ed08a09f.png)


### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* On sites with the VideoPress plugin, add a video
* Confirm that the two new settings are visible and display correctly in different screen sizes
* Test changing the video from the dashboard and confirm that they are saved
* Change the video settings from the block editor and confirm that the changes also show in the dashboard
